src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+find_package(Python 3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 REQUIRED)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+)
+set_target_properties(xtp_binds
+  PROPERTIES
+  CXX_STANDARD 14
+  CXX_STANDARD_REQUIRED ON
+)
+
+
+target_link_libraries(xtp_binds
+ PUBLIC
+ VOTCA::votca_xtp
+)
+
+if(NOT PYTHON_INSTDIR)
+  execute_process(
+    COMMAND
+      ${Python_EXECUTABLE} -c
+      "import distutils.sysconfig as cg; print(cg.get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}', plat_specific=True, standard_lib=False))"
+    OUTPUT_VARIABLE PYTHON_INSTDIR OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif(NOT PYTHON_INSTDIR)
+install(TARGETS xtp_binds LIBRARY DESTINATION ${PYTHON_INSTDIR}/pyxtp)
+
+file(GLOB_RECURSE PYXTP_FILES *.py)
+install(FILES ${PYXTP_FILES} DESTINATION ${PYTHON_INSTDIR}/pyxtp)
```

Comment:
We need to install the examples in the right location!

Reply:
I just try it as it and it is working:
```
python -c "from pyxtp.examples import run_examples; run_examples()"
```

Reply:
I have added a `__init__.py` file to import the examples

Reply:
Ok, is that python common practice?

Reply:
For most python packages the examples are included inside the installation, for instance a normal package has a file tree like:
├── setup.py
├── library
│   ├── __init__.py
│   ├── library.py
│   ├── examples
│   ├──── __init__.py
│   └────example1.py 
└──  tests

But since we don't have a setup and only the examples.py file, I suppose that we can put the library.so and examples.py at the same level

Reply:
I am wondering if examples usually get installed in the module directory.

Reply:
In projects Like Numpy of Pandas, the example is embed into the docstring, for instance the help function for `np.sum` looks like:
```
Parameters
----------
a : array_like
    Elements to sum.
axis : None or int or tuple of ints, optional
.........................
Examples
--------
>>> np.sum([0.5, 1.5])
2.0
>>> np.sum([0.5, 0.7, 0.2, 1.5], dtype=np.int32)
```
But we don't have any Python code into which embed the examples

Reply:
One fedore there are at least some package that install examples in the module directory:
```
$ find python3.9/ -name "*example*"
python3.9/site-packages/numpy/random/_examples
python3.9/site-packages/statsmodels/iolib/__pycache__/stata_summary_examples.cpython-39.pyc
python3.9/site-packages/statsmodels/iolib/__pycache__/stata_summary_examples.cpython-39.opt-1.pyc
python3.9/site-packages/statsmodels/iolib/stata_summary_examples.py
python3.9/site-packages/statsmodels/iolib/tests/results/time_series_examples.dta
python3.9/site-packages/statsmodels/sandbox/distributions/examples
python3.9/site-packages/statsmodels/sandbox/nonparametric/dgp_examples.py
python3.9/site-packages/statsmodels/sandbox/nonparametric/__pycache__/dgp_examples.cpython-39.pyc
python3.9/site-packages/statsmodels/sandbox/nonparametric/__pycache__/dgp_examples.cpython-39.opt-1.pyc
python3.9/site-packages/statsmodels/sandbox/regression/__pycache__/example_kernridge.cpython-39.opt-1.pyc
python3.9/site-packages/statsmodels/sandbox/regression/__pycache__/example_kernridge.cpython-39.pyc
python3.9/site-packages/statsmodels/sandbox/regression/example_kernridge.py
python3.9/site-packages/statsmodels/sandbox/tsa/__pycache__/example_arma.cpython-39.opt-1.pyc
python3.9/site-packages/statsmodels/sandbox/tsa/__pycache__/example_arma.cpython-39.pyc
python3.9/site-packages/statsmodels/sandbox/tsa/example_arma.py
python3.9/site-packages/statsmodels/tsa/vector_ar/tests/example_svar.py
python3.9/site-packages/statsmodels/tsa/vector_ar/tests/__pycache__/example_svar.cpython-39.pyc
python3.9/site-packages/statsmodels/tsa/vector_ar/tests/__pycache__/example_svar.cpython-39.opt-1.pyc
python3.9/site-packages/scipy/io/tests/data/example_1.nc
python3.9/site-packages/scipy/io/tests/data/example_2.nc
python3.9/site-packages/scipy/io/tests/data/example_3_maskedvals.nc
```

Reply:
Then, shall  we keep the examples in the `pyxtp` package? I prefer to have them there so people can have a look.